controllers/v1/reportsController.js

```diff
@@ -3070,5 +3070,38 @@ submissionReport = async function (req) {
     }
   });
 };
+fetch = async function (req) {
+  return new Promise(async function (resolve, reject) {
+    try {
+
+        //  submission observation report
+        if (req.body.submissionId && req.body.observation == true ) {
+
+         let response = await reportsHelper.instaceObservationReport(req);
+         resolve({message:response});
```

Comment:
@borkarsaish65 why we are returning result as message

Reply:
This is how it has been setup @VISHNUDAS-tunerlabs . If I change it to something else it won't show in the response. It is setup this way in the route.js file I think , I didnt change it as it might break other things.

Reply:
@borkarsaish65 can you please check other controllers. I don't think sending data in message key is not  proper
 return resolve({
          message: surveySolutionDetails.message,
          result: surveySolutionDetails.data,
        });
        
        
we used to do  it like this